api.md

```diff
@@ -2767,6 +2767,10 @@ Specifically this sets the `timescaledb.restoring` GUC to `on` and stops any
 background workers which may have been performing tasks until the [`timescaledb_post_restore`](#timescaledb_post_restore)
 fuction is run following the restore. See [backup/restore docs][backup-restore] for more information.
 
+>:WARNING: Note that you do not need to use this function when doing an upgrade
+using `pg_upgrade` and using this function when doing an upgrade could
+cause issue in previous versions of TimescaleDB.
```

Comment:
`using this function when doing an upgrade could cause issue in previous versions of TimescaleDB.` 

I would reword as using this function when doing an upgrade can cause problems in previous versions....

Reply:
Reformulated.